src/content/1.1/code/agda/snippet05.agda

```diff
@@ -1,2 +1,2 @@
-id : ∀ {A} → A → A
+id : ∀ {a} → a → a
```

Comment:
Do you prefer lower case for types because it's more consistent with the Haskell snippets?  I'm okay either way... just curious.